geometry_msgs/Point    position
msi_rover/attitude     attitude
geometry_msgs/Point    lin_velocity
geometry_msgs/Point    ang_velocity
